### ISSUE TITLE
Crashing with AstroidError

### ISSUE THREAD

**Issue by gocarlos** [PyCQA/pylint-django#391]
Need to find out which file is causing the issue, will update the issue as soon as I have it

## Pip Freeze

```txt
aioredis==1.3.1
amqp==5.1.1
anyio==3.6.2
apns @ git+https://github.com/djacobs/PyAPNs.git@833b00f5c58a211d4a00ee4a0c41ca0a0e032270
asgiref==3.6.0
astroid==2.14.1
async-timeout==4.0.2
attrs==22.2.0
autobahn==23.1.1
Automat==22.10.0
bcrypt==4.0.1
billiard==3.6.4.0
boto3==1.17.19
botocore==1.20.112
celery==5.2.7
certifi==2022.12.7
cffi==1.15.1
channels==3.0.3
channels-redis==3.2.0
chardet==4.0.0
click==8.1.3
click-didyoumean==0.3.0
click-plugins==1.1.1
click-repl==0.2.0
constantly==15.1.0
cryptography==3.4.7
daphne==3.0.2
defusedxml==0.7.1
diff-match-patch==20200713
dill==0.3.6
Django==3.2.16
django-bootstrap3==14.2.0
django-cors-headers==3.13.0
django-debug-toolbar==3.2.1
django-filter==2.4.0
django-import-export==2.5.0
django-model-utils==4.1.1
django-redis==5.0.0
django-simple-history==2.12.0
django-storages==1.11.1
django-webpack-loader==0.7.0
djangorestframework==3.14.0
djangorestframework-simplejwt==4.6.0
dnspython==1.16.0
et-xmlfile==1.1.0
eventlet==0.30.2
flower==1.0.0
freezegun==1.1.0
geographiclib==1.52
geopy==2.1.0
googlemaps==4.4.2
greenlet==2.0.2
gunicorn==20.0.4
h11==0.14.0
hiredis==2.2.1
httptools==0.2.0
humanize==4.6.0
hyperlink==21.0.0
idna==2.10
incremental==22.10.0
isort==5.12.0
jdcal==1.4.1
jmespath==0.10.0
kombu==5.2.4
lazy-object-proxy==1.9.0
MarkupPy==1.14
mccabe==0.6.1
mock==4.0.3
model-bakery==1.2.1
msgpack==1.0.4
mt-940==4.23.0
ndg-httpsclient==0.5.1
odfpy==1.4.1
openpyxl==3.0.6
paramiko==2.12.0
pdf2image==1.14.0
Pillow==8.2.0
platformdirs==3.0.0
polib==1.1.0
prometheus-client==0.16.0
prompt-toolkit==3.0.36
psycopg2-binary==2.9.5
pusher==3.2.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycparser==2.21
pyfcm==1.5.1
PyJWT==2.1.0
pylint==2.16.1
pylint-celery==0.3
pylint-django==2.4.4
pylint-plugin-utils==0.7
PyNaCl==1.5.0
pyOpenSSL==21.0.0
pypdf==3.3.0
python-dateutil==2.8.1
python-dotenv==0.21.1
pytz==2022.7
PyYAML==6.0
qrcode==6.1
redis==4.4.2
requests==2.25.1
s3transfer==0.3.7
sentry-sdk==1.9.8
service-identity==21.1.0
six==1.16.0
sniffio==1.3.0
sqlparse==0.4.3
stripe==2.56.0
tablib==3.3.0
toml==0.10.2
tomli==2.0.1
tomlkit==0.11.6
tornado==6.2
Twisted==22.10.0
txaio==23.1.1
typing-extensions==4.4.0
urllib3==1.26.14
uuid==1.30
uvicorn==0.14.0
uvloop==0.17.0
vine==5.0.0
watchgod==0.8.2
wcwidth==0.2.6
websocket-client==1.1.0
websockets==10.4
wrapt==1.12.1
xlrd==2.0.1
xlwt==1.3.0
xmltodict==0.12.0
zope.interface==5.5.2
```

**NOTES:**  make sure you have the latest version of 3rd party packages
like `rest_framework`, `factory`, `model_utils`, etc. before reporting
issues!

## Stacktrace

![image](https://user-images.githubusercontent.com/1331762/217256591-b37cec88-6e59-4bb2-ac25-2c62a8c1f9b7.png)

**Comment by carlio** [comment]
This seems related to an issue fixed a while ago, see https://github.com/PyCQA/pylint-django/issues/358 and check if a newer version of pylint-django will fix the issue for you (currently on 2.5.3, you have 2.4.4)

**Comment by gocarlos** [comment]
yes, thanks @carlio that fixed the issue... there are other crashes but will open issues for that specific ones
